### PR TITLE
INTLY-455: add advanced task: Adding custom services to a walkthrough

### PR DESCRIPTION
**Pull request by pwright**

### CODE REVIEW

File: walkthroughs/writing-walkthroughs/walkthrough.adoc [hunk 1]
```diff
@@ -453,3 +453,87 @@ Check that the step is displayed as a link to the Openshift Console.
...
+
+. Log into the OpenShift CLI using the `oc` on your clipboard.
+
+. Run the following command to list all the service names available from your cluster catalog:
++
+----
+oc get clusterserviceclasses -o custom-columns="displayName:.spec.externalMetadata.displayName, externalName:.spec.externalName"
+----
++
+Typically, service names describe many variations of components.
```
Comment (david-martin) [inline walkthroughs/writing-walkthroughs/walkthrough.adoc]: @finp I think an extra line to explain what you see in the output of above command would be helpful.
There are 2 columns, but what are they?
Also, what column should the user be concerned with for the next step?

File: walkthroughs/writing-walkthroughs/walkthrough.adoc [hunk 2]
```diff
@@ -453,3 +453,87 @@ Check that the step is displayed as a link to the Openshift Console.
...
+{
+  "dependencies": {
+    "repos": [],
+    "serviceInstances": [{
+      "metadata": { "name": "cakephp-mysql-example" },
+      "spec": {
+        "clusterServiceClassExternalName": "cakephp-mysql-example",
+        "clusterServicePlanExternalName": "default"
+      }
+      }]
```
Comment (david-martin) [inline walkthroughs/writing-walkthroughs/walkthrough.adoc]: minor indentation misalignment

File: walkthroughs/writing-walkthroughs/walkthrough.adoc (hunk 1 above)
Comment (aidenkeating) [reply walkthroughs/writing-walkthroughs/walkthrough.adoc]: This is a good point, I also noticed the column names blend in with the rest of the names. Maybe we could do something like?

```
oc get clusterserviceclasses -o custom-columns="Display Name (From Service Catalog):.spec.externalMetadata.displayName, Identifier (clusterServiceClassExternalName):.spec.externalName"
```

The table was created as a mapping between the names the user would know and understand from the Service Catalog UI (left-hand side) and the names/identifiers the user needs when specifying a `clusterServiceClassExternalName` in a ServiceInstance.

File: walkthroughs/writing-walkthroughs/walkthrough.adoc [hunk 3]
```diff
@@ -453,3 +453,87 @@ Check that the step is displayed as a link to the Openshift Console.
...
+. Edit the `walkthrough.adoc` file and use the `{route-<route-name>-host}` pattern, whenever you want the route URL to appear.
+For example, to insert a step that asks the user of the walkthrough to log into the CakePHP app:
++
+----
+. Log into the link:{route-cakephp-mysql-example-host}[CakePHP app].
+----
++
+NOTE: If you want Asciidoc attributes to render values in code blocks, you must start the code block with `[subs="attributes"]`.
+
+. Publish the walkthrough or redeploy the *tutorial-web-app* to pick up changes to your walkthrough
```
Comment (david-martin) [inline walkthroughs/writing-walkthroughs/walkthrough.adoc]: Should this have more information or link to how to publish or redeploy the tutorial-web-app?

**Review by david-martin — CHANGES_REQUESTED** [review]
@finp I've left a few comments.

File: walkthroughs/writing-walkthroughs/walkthrough.adoc (hunk 1 above)
Comment (david-martin) [reply walkthroughs/writing-walkthroughs/walkthrough.adoc]: This looks better.
Referencing the `clusterServiceClassExternalName` from the column for what value to use in  the next step helps

File: walkthroughs/writing-walkthroughs/walkthrough.adoc (hunk 3 above)
Comment (pwright) [reply walkthroughs/writing-walkthroughs/walkthrough.adoc]: added link to  task resources

**Comment by pwright** [comment]
@david-martin @aidenkeating  updated based on comments

**Comment by pwright** [comment]
@david-martin  are you happy with my updates?

**Review by david-martin — APPROVED** [review]

**Pull request merged**